src/SumoCoders/FrameworkCoreBundle/Resources/assets/coffee/framework.flash.coffee

```diff
@@ -3,14 +3,15 @@ class Flash
   add: (message, type) ->
     alertId = Date.now();
 
-    $('body').prepend(
-      '<div class="alert alert-' + type + ' alert-dismissible notification" role="alert" data-alert-id="' + alertId + '">' +
-        '  <div class="container">' +
-        '    <button type="button" class="close" data-dismiss="alert"' +
-        '       title="' + Locale.lbl('Close') + '">' + Locale.lbl('Close') +
-        '    </button>' +
-        '    ' + message +
-        '  </div>' +
+    $('.main-header').append(
+        '<div class="alert alert-fixed alert-' + type + ' alert-dismissible notification" role="status" data-alert-id="' + alertId + '">' +
```

Comment:
to be honest i'd just use the `.main-header .alert` selector, the alert is not 'special', it's just in a special place.